pkg/component/networking/apiserverproxy/apiserver_proxy.go

```diff
@@ -331,7 +331,7 @@ func (a *apiserverProxy) computeResourcesData() (map[string][]byte, error) {
 								ImagePullPolicy: corev1.PullIfNotPresent,
 								Args: []string{
 									fmt.Sprintf("--ip-address=%s", a.values.advertiseIPAddress),
-									"--interface=lo",
+									"--interface=apiserverproxy0",
```

Comment:
As we don't run with `--cleanup=true` the address on the loopback device will not be deleted. Do we need to roll this out in two steps? First enable cleanup and set the interface with the next release?
https://github.com/gardener/apiserver-proxy/blob/5c0731a4a9c96d2ebdd97ddb3b68e1e21c8a8761/cmd/apiserver-proxy-sidecar/main.go#L29